README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
## Improve the README
The `README` is the documentation of the project. It should be as clear as possible. Currently, as it is, a bit vague.